Earthdawn (FASA Official)/sheet.json

```
@@ -0,0 +1,8 @@
+{
+    "html": "Earthdawn.html",
+    "css": "Earthdawn.css",
+    "authors": "Chris Dickey",
+    "roll20userid": "633707",
+    "preview": "Earthdawn.PNG",
+    "instructions": "The Roll20 Earthdawn� (FASA Official) character sheet is optimized for 4th edition, but currently has limited support for other editions./r/r For more information please see https://wiki.roll20.net/Earthdawn_(FASA_Official)"
```

```suggestion
    "instructions": "The Roll20 Earthdawn® (FASA Official) character sheet is optimized for 4th edition, but currently has limited support for other editions./r/r For more information please see [Earthdawn Sheet wiki page](https://wiki.roll20.net/Earthdawn_(FASA_Official))."
```
You could make the wiki page adress a hyperlink this way, the instruction section can use markdown syntax.

---

Thank you. I think I did this unless I messed it up somehow. 
(I made the change and pushed a new commit - Which as far as I can tell it is considering part of the last pull request).

---

Yes, any commit to your branch that you have the PR comiing from automatically updates the PR, ask you see from the new commit under this conversation. 

It's also nice that I finally managed to use the 'code suggestion' feature correctly, it's a bit wonky but has some use.